importer/xmlparser/src/test/java/org/itsallcode/openfasttrace/importer/xmlparser/event/EndElementEventTest.java

```diff
@@ -0,0 +1,41 @@
+package org.itsallcode.openfasttrace.importer.xmlparser.event;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.sameInstance;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import javax.xml.namespace.QName;
+
+import org.itsallcode.openfasttrace.api.core.Location;
+import org.junit.jupiter.api.Test;
+
+import com.jparams.verifier.tostring.ToStringVerifier;
+
+class EndElementEventTest {
+    private static final Location LOCATION = Location.create("path", 42);
+    private static final String URI = "uri";
+    private static final String QNAME = "qname";
+
+    @Test
+    void testToString() {
+        ToStringVerifier.forClass(EndElementEvent.class).verify();
+    }
+
+    @Test
+    void getName() {
+        final QName qName = testee().getName();
+        assertAll(() -> assertThat(qName.getLocalPart(), equalTo("qname")),
+                () -> assertThat(qName.getNamespaceURI(), equalTo("uri")),
+                () -> assertThat(qName.getPrefix(), equalTo("")));
+    }
+
+    @Test
+    void getLocation() {
+        assertThat(testee().getLocation(), sameInstance(LOCATION));
+    }
+
+    private EndElementEvent testee() {
+        return EndElementEvent.create(URI, QNAME, QNAME, LOCATION);
```

Comment:
I am almost certain that the code would be more readable if you hadn't extracted this and used string literals instead of constants. Also, you could use different literals to spice things up. :-)